### PR TITLE
Increasing stack size

### PR DESCRIPTION
**Pull request by ivan-egorov42**
According to #1278.

### CODE REVIEW

**Comment by ivan-egorov42** [comment]
@Graur @mximp pls, have a look

**Review by Graur — APPROVED** [review]
@ivan-egorov42 Looks good to me, thanks!
@yegor256 please, have a look

File: eo-maven-plugin/README.md [hunk 1]
```diff
@@ -124,4 +124,9 @@ Maven option `-U` (see [Maven CLI docs](https://maven.apache.org/ref/3.1.0/maven
 mvn -U clean install
 ```
 
+If you want to build your project or run your tests which use eo-maven-plugin, you can change stack size for your purposes by stack-size option:
 
+```
+mvn clean install -Pqulice -D.stack-size=1M
```
Comment (mximp) [inline eo-maven-plugin/README.md:130]: @ivan-egorov42 why do we need to introduce new property? 
Can't we just run ```mvn clean install -Pqulice -Xss1M```?
If I'm not mistaken Maven will recognise it.

Comment (mximp) [reply eo-maven-plugin/README.md:130]: got it - it was made to introduce increased default value

**Review by mximp — APPROVED** [review]

File: eo-maven-plugin/README.md (hunk 1 above)
Comment (ivan-egorov42) [reply eo-maven-plugin/README.md:130]: `-Xss` is jvm flag, so we need to give it to surefire plugin by options of `mvn`

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**